.github/workflows/ci.yml

```diff
@@ -52,6 +52,26 @@ jobs:
       run: |
         choco install -y doxygen.install
 
+    - name: Install Catch2 (Linux + MacOS)
+      if: runner.os != 'Windows'
+      run: |
+        git clone -b v2.13.3 https://github.com/catchorg/Catch2.git
```

Comment:
version numbers could be env vars to avoid having to update them in multiple places, e..g.
`env: CATCH_VERSION: 'v2.13.3'`

Reply:
I had problems with portably dereferencing an environment variable yesterday when trying the `$SUDOCMD` solution. Have you used that before?

Reply:
yes, see e.g. https://github.com/spatial-model-editor/sme_deps_common/blob/master/.github/workflows/release.yml (although this example uses MSYS2 in windows, so if it was windows that was giving you problems let's just leave this for a future PR)

Reply:
It was Windows indeed, I will add an issue as a reminder.